### PR TITLE
[devtools-translate] Korean - What's New in DevTools (Chrome 107)

### LINKED ISSUE #3758: [devtools-translate] Korean - What's New in DevTools (Chrome 107)
Post: https://developer.chrome.com/blog/new-in-devtools-107/
Translated by: Oct-11

Translator: @techhtml
Reviewers: @cwdoh, @TORU0239

### PR DESCRIPTION
**Pull request by techhtml**
Fixes #3758

Changes proposed in this pull request:
- Translate What's new in DevTools 107 into Korean

### CODE REVIEW

**Comment by jecfish** [comment]
Swift! Over to @TORU0239 for review!

File: site/ko/blog/new-in-devtools-107/index.md [hunk 1]
```diff
@@ -14,107 +14,98 @@ tags:
...
 
 
-<!-- ## Start DOM search after pressing `Enter` {: #search-type } -->
+## `Enter` 키를 눌러 DOM 검색 시작 {: #search-type }
 
-<!-- You can now disable the **Search as you type** setting to always start DOM search after pressing <kbd>Enter</kbd>.  -->
+<kbd>Enter</kbd> 키를 누른 후 항상 DOM 검색을 시작하는 **Search as you type** 설정을 비활성화할 수 있습니다.
 
-<!-- In the **Elements** panel, toggle the search bar with <kbd>Control</kbd> or <kbd>Command</kbd> + <kbd>F</kbd>. As you type a query in the search textbox, the DOM tree will jump to the first matching element and highlight it by default.  -->
+**요소** 패널에서 <kbd>Control</kbd> 이나 <kbd>Command</kbd> + <kbd>F</kbd> 키를 통해 검색바를 전환합니다. 검색 입력창에서 쿼리를 입력하면 기본 동작으로 DOM 트리가 첫번째로 일치한 요소로 이동하고 하이라이트합니다.
```
Comment (TORU0239) [inline site/ko/blog/new-in-devtools-107/index.md:79]: How about '강조 표시합니다' for '하이라이트합니다'?

File: site/ko/blog/new-in-devtools-107/index.md [hunk 2]
```diff
@@ -14,107 +14,98 @@ tags:
...
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/029ac9db0b7e7d08945bcf7a16b407bde50183a1 #}
 
 Chromium issues: [1280398](https://crbug.com/1280398), [1226363](https://crbug.com/1226363)
 
 
-<!-- ## Highlight C/C++ objects in the Memory Inspector {: #memory } -->
+## 메모리 인스펙터에서 C/C++ 객체 하이라이팅 {: #memory }
 
-<!-- The [Memory Inspector](/docs/devtools/memory-inspector/) highlights all the bytes of a C/C++ memory object. -->
+[메모리 인스펙터](/docs/devtools/memory-inspector/) 에서 모든 C/C++ 메모리 객체의 바이트를 하이라이팅합니다.
```
Comment (TORU0239) [inline site/ko/blog/new-in-devtools-107/index.md:47]: I noticed that this paragraph is illustrated with blue colored highlights.
So, you might want to translate like this,  '메모리 객체의 바이트를 강조표시합니다.'

File: site/ko/blog/new-in-devtools-107/index.md [hunk 3]
```diff
@@ -14,107 +14,98 @@ tags:
...
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/d061128ff63a97ab2c6c0d2b5e655e6fcbed829c #}
 
 Chromium issues: [1335274](https://crbug.com/1335274), [174309](https://crbug.com/174309)
 
 
-<!-- ## Toggle light and dark themes with keyboard shortcut {: #toggle-themes } -->
+## 키보드 단축키로 라이트 / 다크 테마 전환하기 {: #toggle-themes }
 
-<!-- Configure a keyboard shortcut to toggle [light and dark themes](/docs/devtools/rendering/emulate-css/#emulate-css-media-feature-prefers-color-scheme) conveniently. By default, the action doesn’t map to any keyboard shortcut. -->
+편리하게 [라이트 / 다크 테마](/docs/devtools/rendering/emulate-css/#emulate-css-media-feature-prefers-color-scheme) 를 전환하는 키보드 단축키를 설정할 수 있습니다. 기본적으로는 이 전환 액션에 대한 키보드 단축키가 설정되어있지 않습니다.
```
Comment (TORU0239) [inline site/ko/blog/new-in-devtools-107/index.md:35]: Need a whitespace like '단축키가 설정되어 있지 않습니다.'

File: site/ko/blog/new-in-devtools-107/index.md [hunk 4]
```diff
@@ -14,107 +14,98 @@ tags:
...
 
 {% include 'partials/devtools/ko/banner.md' %}
 
-<!-- Translation instructions:
-  1. Remove the "draft: true" tag above when submitting PR
-  2. Provide translations under each of the English commented original content
-  3. Translate the "description" tag above
-  4. Translate all the <img> alt text
-  5. Update the whats-new.md file -->
+## DevTools에서 키보드 단축키 커스터마이징 {: #shortcuts }
```
Comment (TORU0239) [inline site/ko/blog/new-in-devtools-107/index.md:20]: I am a little curious which will be better, '커스터마이징' or '사용자 정의' :)

**Review by TORU0239 — COMMENTED** [review]
Done, back to you @techhtml

**Review by jecfish — APPROVED** [review]

**Pull request merged**